src/idl_gen_swift.cpp

```diff
@@ -162,13 +162,16 @@ class SwiftGenerator : public BaseGenerator {
 
   bool generate() {
     code_.Clear();
+    code_.SetValue("IMPLEMENTATION", parser_.opts.swift_implementation_only
+                                         ? "@_implementationOnly "
+                                         : "");
     code_.SetValue("ACCESS", "_accessor");
     code_.SetValue("TABLEOFFSET", "VTOFFSET");
     code_ += "// " + std::string(FlatBuffersGeneratedWarning());
     code_ += "// swiftlint:disable all";
     code_ += "// swiftformat:disable all\n";
     if (parser_.opts.include_dependence_headers || parser_.opts.generate_all)
-      code_ += "import FlatBuffers\n";
+      code_ += "{{IMPLEMENTATION}}import FlatBuffers\n";
```

Comment:
I don't see this edit in the generated code. Can we make sure that it is included?

Reply:
I believe we can't really use it in the code generation since it a special case that was requested in https://github.com/google/flatbuffers/issues/7201

Reply:
can we just add  a new line in `scripts/generate_code.py` to try this option?